EIPS/eip-####.md

```diff
@@ -0,0 +1,182 @@
+---
```

Comment:
```suggestion
---
eip: 7658
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.